src/Doctrine/Odm/Filter/AbstractFilter.php

```diff
@@ -56,11 +60,16 @@ public function apply(Builder $aggregationBuilder, string $resourceClass, ?Opera
      */
     abstract protected function filterProperty(string $property, $value, Builder $aggregationBuilder, string $resourceClass, ?Operation $operation = null, array &$context = []): void;
 
-    protected function getManagerRegistry(): ManagerRegistry
+    protected function getManagerRegistry(): ?ManagerRegistry
     {
         return $this->managerRegistry;
     }
 
+    public function setManagerRegistry(?ManagerRegistry $managerRegistry): ?ManagerRegistry
```

Comment:
we need to add an interface for this and check that a filter implements the interface before calling `setManagerRegistry`

Reply:
Don't all filters implement `AbstractFilter.php`?

Reply:
no

Reply:
I created the `ManagerRegistryConfigurableInterface` interface, implemented the interface in `AbstractFilter` and added a tester in `ParameterExtension` for `Odm` + `ORM`. What do you think?